pulp_ansible/app/serializers.py

```diff
@@ -85,6 +87,16 @@ class Meta:
         model = AnsibleRepository
 
 
+class AnsibleRepositorySyncURLSerializer(RepositorySyncURLSerializer):
+    """
+    Serializer for Ansible RepositorySyncURL.
+    """
+
+    page_size = serializers.IntegerField(
+        help_text=_("The total of results per page."), default=PAGE_SIZE,
```

Comment:
Does the default show in the openAPI schema for users?

Reply:
not sure

Reply:
it does
![image](https://user-images.githubusercontent.com/17153022/84186392-da0ea380-aa66-11ea-961f-b14449f87877.png)